apps/communications/dialer/index.html

```diff
@@ -170,43 +171,43 @@ <h1 data-l10n-id="callLog"></h1>
             <section id="keypad" dir="ltr">
               <div class="keypad-row">
                 <div class="keypad-key" role="key" data-value="1">
-                  <span>1</span>
+                  <span data-l10n-numerals>1</span>
```

Comment:
This has to change if it's gonna be a standalone library, maybe to something like `data-intl-numerals`.